nodebuilder/testing.go

```diff
@@ -46,7 +46,7 @@ func TestNodeWithConfig(t *testing.T, tp node.Type, cfg *Config, opts ...fx.Opti
 	require.NoError(t, err)
 	cfg.Core.IP = ip
 	cfg.Core.RPCPort = port
-	cfg.RPC.Port = "0"
+	cfg.RPC.Port = "26655"
```

Comment:
?

Reply:
25565

Reply:
https://www.quora.com/Why-did-Minecraft-choose-port-25565-as-default

Reply:
duplicate of https://github.com/celestiaorg/celestia-node/pull/1195#discussion_r1009154625, resolving